### ISSUE TITLE
GitHub workflow to put a message once Journey is merged.

### ISSUE THREAD

**Issue by KendallDoesCoding** [collab-community/journey-book#115]
It would be epic, if for **ADDING JOURNEY PR'S** there was a GitHub action that once the PR is merged, would drop a message saying:

Woo-Hoo, your Journey is now merged.
Check it out on our website - {https://collab-community.github.io/journey-book/#/journeys/{username}}

Is this possible @vikasganiga05? If yes, please find a workflow that does it, or let me know if you can code a workflow to do so.

**Comment by vikasganiga05** [comment]
Good idea but this will be complicated because the workflow will run for every PR. I don't know if is there a way to run for particular PR.

**Comment by KendallDoesCoding** [comment]
> Good idea but this will be complicated because the workflow will run for every PR. I don't know if is there a way to run for particular PR.

Run only if files changes are having the BOTH the extension, .md, .json coz the journey PR will have to have that?

**Comment by vikasganiga05** [comment]
> > Good idea but this will be complicated because the workflow will run for every PR. I don't know if is there a way to run for particular PR.
> 
> Run only if files changes are having the BOTH the extension, .md, .json coz the journey PR will have to have that?

Yes, I need to do some research on that. I will try to complete it as soon as possible. If anyone found a solution please share.

**Comment by KendallDoesCoding** [comment]
> > > Good idea but this will be complicated because the workflow will run for every PR. I don't know if is there a way to run for particular PR.
> > 
> > 
> > Run only if files changes are having the BOTH the extension, .md, .json coz the journey PR will have to have that?
> 
> Yes, I need to do some research on that. I will try to complete it as soon as possible. If anyone found a solution please share.

Ok cool!
